CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
 # Unreleased
+ 
+## New features
+
+### Replace back link placeholder URLs with javascript
+
+Previously, users had to manually replace the `href` for each back link in the page templates.
+
+Now, we use JavaScript to make the back link take you to the previous page by default.
```

Comment:
Before, you had to manually replace the `href` attribute for each back link in the page templates.